extras/ribbon.js

```diff
@@ -0,0 +1,98 @@
+/**
+ * @license
+ * Copyright 2015 Petr Shevtsov (petr.shevtsov@gmail.com)
+ * MIT-licensed (http://opensource.org/licenses/MIT)
+ *
+ * Plugin options:
+ *
+ *  `data`: Array of numeric values (0-1) corresponding to the position
+ *  in the pallete interval.
+ *
+ *  `parser`: Function (`function (data, dygraph)`) returning the array of numeric
+ *  values. Function arguments: raw data, dygraph instance.
+ *
+ *  `options`: Object with the following properties:
+ *
+ *    `palette`: Colors Array. Default: ["transparent", "#ef2929", "#8ae234"]
+ *
+ *    `height`: Value (0-1) representing the height of the ribbon: 1 - full height,
```

Comment:
it's interesting that the x-values of the ribbons are tied to the data whereas the y-values are not (the ribbon doesn't move with y-panning and y-zooming). Why did you choose to do it this way?